src/frontend/src/pages/login/login.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+// for fields
+import TextField from '@material-ui/core/TextField';
+
+// for submit
+import Button from '@material-ui/core/Button';
+import {
+    Link,
+} from "react-router-dom"
```

Comment:
```suggestion
import { Link } from "react-router-dom"
```
This can be on a single line. Also, the comment is unnecessary.